pkg/cloud/volume_limits.go

```diff
@@ -131,215 +131,337 @@ func GetNVMeInstanceStoreVolumesForInstanceType(it string) int {
 	return 0
 }
 
-// / https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes
+// / https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-volumes.html
 // / IMDS does not provide NVMe instance store data; we'll just list all instances here
 // / TODO: See if we can get these values from DescribeInstanceTypes API
+// Generated by command, requires aws cli and jq:
+// aws ec2 describe-instance-types --region us-east-1 --filters "Name=instance-storage-supported,Values=true" --query "InstanceTypes[].[InstanceType, InstanceStorageInfo]" | jq -r 'map("\"" + .[0] + "\": " + (.[1].Disks | map(.Count) | add | tostring) + ",") | .[]' | sort
 var nvmeInstanceStoreVolumes = map[string]int{
```

Comment:
Is there a reason we don't support [hpc6id.32xlarge instance?](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-volumes.html#instance-store-vol-hpc)

4 instance store volumes.

Reply:
Hmm, it's only available in us-east-2 so when I go looking in us-east-1 I don't find it. Be ready for the insane kludge of v2

Reply:
Wow that's unfortunate. Thanks for letting us know the reason 😅